backend-rust/src/graphql_api.rs

```diff
@@ -3075,15 +3077,17 @@ struct Account {
     /// The total number of transactions this account has been involved in or
     /// affected by.
     num_txs:           i64,
+    delegated_restake_earnings: Option<bool>,
+    delegated_target_baker_id: Option<i64>
     // Get baker information if this account is baking.
     // baker: Option<Baker>,
-    // delegation: Option<Delegation>,
```

Comment:
Already presented on the object directly and not really a reason to abtract it into another object. I am doing that on the graphql though for backwards compatibility.